src/main/java/org/kainos/ea/api/JobRoleService.java

```diff
@@ -30,4 +35,8 @@ public List<JobRole> getJobRoleById(int id) throws ActionFailedException, JobRol
         return jobRole;
     }
 
+    public String deleteJobRoleById(int id) throws ActionFailedException, ValidationException {
+        jobRequestValidator.checkJobRoleIdExists(databaseConnector, jobRoleDao, id);
```

Comment:
Instead of having this as a validator, could you not just check the job role exists (unless you don't have the functionality to get an individual job role)

Reply:
All Fixed! :)